index.html

```diff
@@ -387,6 +388,45 @@ <h3> <span class="glyphicon glyphicon-exclamation-sign"></span> Legal and Privac
  </div>
 </section>
 
+<section>  
+  <div class="content" id="cvrm">
+    <h1>Predictive Policing</h1>
+    <p>Since 2012, the CPD has used a statistical model to predict which individuals are most likely to be involved in shooting incidents. The model, developed in collaboration with the Illinois Institute of Technology, was initially called the Strategic Subject List (SSL) and was later revised and renamed the Crime and Victimization Risk Model (CVRM). This model was quietly <a id="cvrmlink" href="https://igchicago.org/2020/01/23/advisory-concerning-the-chicago-police-departments-predictive-risk-models/">decommissioned</a> by the CPD in 2019. </p>
+    <div class="row">
+    	<div class="col-md-6"></div>
+        <div class="col-md-6"><img src="img/CVRMimage.png" alt="Socmint" class="img-rounded"></div>
+    </div>
+ </div>
+</section>
+
+<section class="nonheading">  
+  <div class="content">
+  	<div class="row">
+    <h3> <span class="glyphicons glyphicon glyphicon-map-marker"></span> Where</h3> 
+    <p>The CVRM was meant to play a key role in CPD’s new <a href="https://www.rand.org/pubs/research_reports/RR3242.html">Strategic Decision and Support Centers (SDSCs)</a>, real time crime centers which are now located in all but two of Chicago’s 22 districts. Officers could also view at risk individuals identified by the CVRM on a phone app.</p>
```

Comment:
I would say "real time crime centers"

Reply:
I said that the SDSCs are real time crime centers. Is it too wordy?